ui/material.py

```diff
@@ -107,6 +107,27 @@ def draw(self, context):
                 op.preset = preset
 
 
+class LUXCORE_PT_settings(MaterialButtonsPanel, Panel):
+    bl_label = "Settings"
+    bl_context = "material"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(cls, context):
+        engine = context.scene.render.engine
+        return engine == "LUXCORE"
+
+    def draw(self, context):
+        layout = self.layout
+
+        mat = context.material
+        cmat = mat.cycles
+
+        split = layout.split()
+        col = split.column()
+        col.label("Viewport Color:")
+        col.prop(mat, "diffuse_color", text="")
```

Comment:
I would simplify this:
```python
layout.prop(mat, "diffuse_color", text="Viewport Color")
```
It also looks nicer:
![2019-03-14_09-32-27](https://user-images.githubusercontent.com/1584761/54342257-491b0380-463c-11e9-84c5-23da891edb8a.png)